apps/adminer.sh

```diff
@@ -55,9 +55,14 @@ curl_to_dir "https://raw.githubusercontent.com/Niyko/Hydra-Dark-Theme-for-Admine
 ln -s "$ADMINERDIR"/latest.php "$ADMINERDIR"/adminer.php
 
 cat << ADMINER_CREATE > "$ADMINER_CONF"
-Listen 8443
+ <VirtualHost *:80>
+     RewriteEngine On
+     RewriteRule ^(.*)$ https://%{HTTP_HOST}$1:9443 [R=301,L]
+ </VirtualHost>
```

Comment:
This would only work, if I access the server on http://ipaddress
and not on http://ipaddress:9443
So it unfortunately doesn't solve the issue but I guess there is just no solution to this.
BTW: I would remove that part again because you might want to  access your nextcloud on the default ports and not adminer

Reply:
Hmm don't have time to fix that now. Tried some different stuff and would like to avoid using `.htaccess` Maybe we can figure it out later.